index.html

```diff
@@ -34,25 +35,25 @@ <h1>Game</h1>
             </div>
         </div>
         <div id="gameScreen">
-            <script src="/socket.io/socket.io.js"></script>
-            <h1>Your game code is: 
-                <span id = "dispalyGameCode"></span>
-            </h1>
-            <div class='' id="createUsername">
-                <input type="text" placeholder="Enter username" id="usernameInput">
+            <div id="playMenu" style="position: absolute; z-index:2">
```

Comment:
no change requested just question Is it just hiding the div so it disappears after the game started?